### ISSUE TITLE
Getting subscription counts

### ISSUE THREAD

**Issue by tf** [pushpad/pushpad-ruby#4]
I'd like to add a way to use the new `X-Total-Count` header of the `GET /projects/PROJECT_ID/subscriptions` REST API endpoint to make count queries. The API could look like this:
```ruby
Pushpad::Subscription.count(project_id: 5) # => 100
Pushpad::Subscription.count(project_id: 5, uids: ['user1']) # => 2
Pushpad::Subscription.count(project_id: 5, tags: ['sports']) # => 10
Pushpad::Subscription.count(project_id: 5, tags: 'sports && travel') # => 5
Pushpad::Subscription.count(project_id: 5, uids: ['user1'], tags: 'sports && travel') # => 1
```
`project_id` would be a required option if it has not been defined globally.

**Comment by collimarco** [comment]
Ok perfect, you can add it

Thanks

**Comment by collimarco** [comment]
Feature merged in #5
